controllers/utils/constants.go

```diff
@@ -1,15 +1,29 @@
 package utils
 
-// IBUName defines the valid name of the CR for the controller to reconcile
+import (
+	"path/filepath"
+)
+
 const (
+	LcaNamespace string = "openshift-lifecycle-agent"
```

Comment:
Cleanup for another PR. Can we use the one already defined in https://github.com/openshift-kni/lifecycle-agent/blob/main/internal/common/consts.go#L51 or we could pass this in as an environment variable.

Reply:
Didn't realize this one had gotten created. I was originally getting it from an env var, since it wasn't in an accessible constant, so thought I'd create one

Reply:
Done